rwgame/RWGame.cpp

```diff
@@ -67,14 +68,20 @@ RWGame::RWGame(int argc, char* argv[])
 	;
 
 	po::variables_map vm;
-	po::store(po::parse_command_line(argc, argv, desc), vm);
-	po::notify(vm);
+	try
+	{
+		po::store(po::parse_command_line(argc, argv, desc), vm);
+		po::notify(vm);
+	}
+	catch (po::unknown_option& ex)
+	{
+		help = true;
+	}
 
-	if( vm.count("help") )
+	if( help || vm.count("help") )
 	{
-		// TODO: This is a hack
 		std::cout << desc << std::endl;
-		throw std::runtime_error("Terminate");
+		throw std::invalid_argument("");
```

Comment:
nit: shouldn't this only happen in the `help` case?
I didn't test yet, but it looks like `./rwgame --help` would cause invalid argument? Is that standard? Here is a little `cat` (meow) experiment:

```
$ cat  --error; echo $?
cat: unrecognized option '--error'
Try 'cat --help' for more information.
```
Returns 1
```
$ cat  --help; echo $?
Usage: cat [OPTION]... [FILE]...
Concatenate FILE(s) to standard output.
...
Full documentation at: <http://www.gnu.org/software/coreutils/cat>
or available locally via: info '(coreutils) cat invocation'
```
Returns 0